beluga_amcl/src/amcl_node.cpp

```diff
@@ -923,10 +924,16 @@ void AmclNode::laser_callback(ExecutionPolicy&& exec_policy, sensor_msgs::msg::L
   const auto update_start_time = std::chrono::high_resolution_clock::now();
   const auto filter_updated = particle_filter_->update_filter(
       exec_policy, odom_to_base_transform,
-      utils::make_points_from_laser_scan(
-          *laser_scan, base_to_laser_transform, static_cast<std::size_t>(get_parameter("max_beams").as_int()),
-          static_cast<float>(get_parameter("laser_min_range").as_double()),
-          static_cast<float>(get_parameter("laser_max_range").as_double())),
+      beluga_ros::LaserScan{
```

Comment:
Note that I didn't integrate the LaserScan class to the particle filter interface yet. That will come in a follow-up PR.